shared_memory/main.cpp

```diff
@@ -167,9 +167,6 @@ int main(int argc, char **argv)
 
 
 	content_service.SetCurrentExpansion(RuleI(Expansion, CurrentExpansion));
-	content_service.SetDatabase(&database)
```

Comment:
This shouldn't have been removed

Reply:
I figured this wasn't needed anymore since the passing in of the content flags were no longer used.

Reply:
It should still be there because that is the uniform initializer logic that loads both the flags and sets the expansion context.

If we change the source of the current expansion to be pulled from somewhere it will be handled by the loader functions in the content service rather than being passed in 

Doesn't hurt to keep it

Reply:
So this would end up making it always need to do a hotfix, which I assume is ok?